zulipterminal/model.py

```diff
@@ -702,6 +703,55 @@ def get_other_subscribers_in_stream(self, stream_id: Optional[int]=None,
                     if stream['name'] == stream_name
                     if sub != self.user_id]
 
+    def get_tidied_user_info(self, user_id: int) -> Dict[str, Any]:
+        data = self._user_info_by_id(user_id)
```

Comment:
Minor: we could emphasize that this is "raw" data by naming this eg. `api_user_data`, or something shorter?

Reply:
`api_user_data` sgtm :+1: